Save images to disk

trekkie1701c opened the pull request:
I got tired of manually saving images all the time, so:

Add image saving to whatever the current working directory is.  Format would be text prompt, version of dall-e used, and then finally the time (to avoid overwrites).

I suppose one could comment out line 31 to disable it, and either have it disabled by default and require uncommenting or enabled by default and require commenting to disable.

haaakon commented:
It could also be a check box in the web form maybe? Which is default checked.

VeXHarbinger commented:
Ahh I didn't see your branch and I wrote a similar update to allow for images to download on click so you could pick and choose.  
https://github.com/saharmor/dalle-playground/pull/37
I like that you appended with the date though to block overwrites

trekkie1701c commented:
Added some pull requests from others to my repo for my own testing and forgot to swap off the main branch so they've all shown up here as a bit of a mess.  I've reverted them so it should be just the four lines of code that I'd originally added.  Apologies for the mess and I'll be a bit more cautious with it in the future.
